container.go

```diff
@@ -192,6 +192,10 @@ func GetContainers(q ComputeSystemQuery) ([]ContainerProperties, error) {
 	)
 	err = hcsEnumerateComputeSystems(query, &computeSystemsp, &resultp)
```

Comment:
Did you mean this to be the line below? `processHcsResult` currently is just a passthrough for the error, it doesn't do any checking. We were never actually checking the error returned here.